src/v/cluster/data_migration_worker.h

```diff
@@ -48,7 +48,7 @@ class worker : public ss::peering_sharded_service<worker> {
     abort_partition_work(model::ntp&& ntp, id migration_id, state sought_state);
 
 private:
-    struct ntp_state_t {
+    struct mntp_state_t {
```

Comment:
consider adding a comment explaining the name, it was obvious for `ntp` now it is not